package.json

```diff
@@ -96,8 +96,10 @@
     "@ethersproject/transactions": "5.7.0",
     "@ethersproject/units": "5.7.0",
     "@ethersproject/wallet": "5.7.0",
-    "@ledgerhq/hw-app-eth": "6.35.3",
+    "@ledgerhq/evm-tools": "1.2.0",
+    "@ledgerhq/hw-app-eth": "6.38.0",
     "@ledgerhq/hw-transport-webhid": "6.28.2",
```

Comment:
@BrodyHughes Is there a reason we dropped hw-transport-webhid to `6.28`? 6.29 is compatible with the Flex devices

Reply:
no reason no. i don't remember specifying 28 when i bumped. i imagine bumping to 29 is fine.

Reply:
hopefully this fixes e2e and bumping is easy now